### ISSUE TITLE
OLM installation references non-existing images

### ISSUE THREAD

**Issue by ctron** [openshift/openshift-ansible#10614]
#### Description

When setting `openshift_enable_olm=true`, a deployment is being created with non-existing images.

##### Version

Please put the following version information in the code block
indicated below.

* Your ansible version per `ansible --version`

~~~
# ansible --version
ansible 2.6.5
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Jul 13 2018, 13:06:57) [GCC 4.8.5 20150623 (Red Hat 4.8.5-28)]
~~~

* The output of `git describe`

~~~
# git describe
openshift-ansible-3.11.39-1
~~~

##### Steps To Reproduce
1. Set `openshift_enable_olm=true`
2. Run `ansible-playbook -u root -i inventory.txt openshift-ansible/playbooks/prerequisites.yml && ansible-playbook -u root -i inventory.txt openshift-ansible/playbooks/deploy_cluster.yml`

##### Expected Results

OpenShift gets set up with OLM ready to be used.

##### Observed Results
Describe what is actually happening.

The `operator-lifecycle-manager` projects gets created. However the two deployments do get created, referencing images that do not exists:

e.g for `olm-operator`: `image: quay.io/coreos/olm` -> which references `:latest`. However that tag does not exist: https://quay.io/repository/coreos/olm?tab=tags

Which results in:

~~~
Failed to pull image "quay.io/coreos/olm": rpc error: code = Unknown desc = Tag latest not found in repository quay.io/coreos/olm
~~~

**Comment by vrutkovs** [comment]
@ecordell seems OLM image is not versioned - is it done on purpose?

**Comment by gacopl** [comment]
Any plans to update OLM? the one provided is missing clusterPermission CSV features, havent seen any work for OLM in repo since september are there any plans to continue work on it ?

**Comment by mlbiam** [comment]
bump...

**Comment by SPFZ** [comment]
This issue should be closes because #11469 was merged.
